core_tic_tac_toe/game_config.py

```diff
@@ -8,64 +8,78 @@
 
 class GameConfig:
 
-    def __init__(self, board, display, game_logic, input_validator, menu):
-        self.board = board
+    def __init__(self, display, game_logic, input_validator, menu):
         self.display = display
         self.game_logic = game_logic
         self.input_validator = input_validator
         self.menu = menu
-        self.game_options = "1. Human vs Human \n2. Human vs Easy Computer\n3. Human vs Medium Computer\n4. Human vs Hard Computer"
+        self.player_game_options = "1. Human vs Human \n2. Human vs Easy Computer\n3. Human vs Medium Computer\n4. Human vs Hard Computer"
+        self.board_option = "What size board would you like to play on? Enter a value between 3 to 9 (e.g 3 will generate a 3X3 board)"
 
-    def prepare_players(self, player_selection):
+    def prepare_players(self, player_selection, board):
 
         if player_selection == "1":
-            self.__prepare_human_players()
+            return self.__prepare_human_players(board)
```

Comment:
What's the difference between not using and using `return` in these spots?

Reply:
Each of the methods return an array of players, which this method ultimately needs to return to the `prepare_game` method. Instead of using multiple return statements, which can seem confusing. I can assign the evaluation of each of the methods to a variable, which I then return at the end of the `prepare_players` method

Reply:
I'm not suggesting a change. I don't find the multiple `return` statements confusing. I'm just asking why you added the `return` statements. Presumably, the code worked before without them.